great-idea-website/index.html

```diff
@@ -15,49 +15,70 @@
 </head>
 
   <body>
+  <nav class="navigation">
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+  </nav>
+  
+<header class="header">
```

Comment:
The header could also include the <nav> section, though it's not mandatory that it does.